src/falco/config.vsh.yaml

```
@@ -0,0 +1,197 @@
+          your plots may end up a ridiculous size. You 
+          have been warned!
+      - name: --contaminents
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          the list of contaminants to screen 
+          overrepresented sequences against. The file 
+          must contain sets of named contaminants in 
+          the form name[tab]sequence. Lines prefixed 
+          with a hash will be ignored. Default: 
+          /tmp/falco-1.2.2/Configuration/contaminant_list.txt
+      - name: --adapters
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          the list of adapter sequences which will be 
+          explicity searched against the library. The 
+          file must contain sets of named adapters in 
+          the form name[tab]sequence. Lines prefixed 
+          with a hash will be ignored. Default: 
+          /tmp/falco-1.2.2/Configuration/adapter_list.txt
+      - name: --limits
+        type: file
+        description: |
+          Specifies a non-default file which contains 
+          a set of criteria which will be used to 
+          determine the warn/error limits for the 
+          various modules. This file can also be used 
+          to selectively remove some modules from the 
+          output all together. The format needs to 
+          mirror the default limits.txt file found in 
+          the Configuration folder. Default: 
+          /tmp/falco-1.2.2/Configuration/limits.txt
+      - name: -subsample
+        alternatives: [-s]
+        type: boolean_true
+        description: |
+          [Falco only] makes falco faster (but 
+          possibly less accurate) by only processing 
+          reads that are multiple of this value (using 
+          0-based indexing to number reads).
+      - name: -bisulfite
+        alternatives: [-b]
+        type: boolean_true
+        description: |
+          [Falco only] reads are whole genome 
+          bisulfite sequencing, and more Ts and fewer 
+          Cs are therefore expected and will be 
+          accounted for in base content.
+      - name: -reverse_complliment
+        alternatives: [-r]
+        type: boolean_true
+        description: |
+          [Falco only] The input is a 
+          reverse-complement. All modules will be 
+          tested by swapping A/T and C/G
+
+  - name: Output arguments
+    arguments:
+      - name: --outdir
+        alternatives: [-o]
+        type: file
+        direction: output
+        description: |
+          Create all output files in the specified 
+          output directory. FALCO-SPECIFIC: If the 
+          directory does not exists, the program will 
+          create it. If this option is not set then 
+          the output file for each sequence file is 
+          created in the same directory as the 
+          sequence file which was processed.
```

Is this still relevant in the context of viash?
```suggestion
          output directory. FALCO-SPECIFIC: If the 
          directory does not exists, the program will 
          create it. If this option is not set then 
          the output file for each sequence file is 
          created in the same directory as the 
          sequence file which was processed.
```

---

Yes, I think so. If you run falco on multiple inputs, the results are stored next to the input data if you don't specify an output directory.

---

In this case, will nextflow know where to grab the files?